execution/gethexec/sequencer.go

```diff
@@ -117,6 +117,34 @@ var DefaultTimeboostConfig = TimeboostConfig{
 	RedisUrl:                  "unset",
 }
 
+type DangerousConfig struct {
+	Timeboost TimeboostConfig `koanf:"timeboost"`
+}
+
+type TimeboostConfig struct {
+	Enable                    bool          `koanf:"enable"`
+	AuctionContractAddress    string        `koanf:"auction-contract-address"`
+	AuctioneerAddress         string        `koanf:"auctioneer-address"`
+	ExpressLaneAdvantage      time.Duration `koanf:"express-lane-advantage"`
+	SequencerHTTPEndpoint     string        `koanf:"sequencer-http-endpoint"`
+	EarlySubmissionGrace      time.Duration `koanf:"early-submission-grace"`
+	MaxQueuedTxCount          int           `koanf:"max-queued-tx-count"`
+	MaxFutureSequenceDistance uint64        `koanf:"max-future-sequence-distance"`
+	RedisUrl                  string        `koanf:"redis-url"`
+}
+
+var DefaultTimeboostConfig = TimeboostConfig{
```

Comment:
why is this missing from our default config? Did we mess up something in the merge?

Reply:
That is actually a very good question ...